python/src/ops.cpp

```diff
@@ -2931,20 +2931,41 @@ void init_ops(py::module_& m) {
       "load",
       &mlx_load_helper,
       "file"_a,
+      "format"_a = none,
       py::pos_only(),
       py::kw_only(),
       "stream"_a = none,
       R"pbdoc(
-        load(file: str, /, *, stream: Union[None, Stream, Device] = None) -> Union[array, Dict[str, array]]
+        load(file: str, format: Optional[str] = None, /, *, stream: Union[None, Stream, Device] = None) -> Union[array, Dict[str, array]]
 
         Load array(s) from a binary file in ``.npy`` or ``.npz`` format.
 
         Args:
             file (file, str): File in which the array is saved
-
+            format (str, optional): Format of the file. If ``None``, the format
+              is inferred from the file extension. Supported formats: npy, npz, and safetensors. (default: ``None``)
         Returns:
             result (array, dict): The loaded array if ``.npy`` file or a dict mapping name to array if ``.npz`` file
       )pbdoc");
+  m.def(
+      "save_safetensor",
+      &mlx_save_safetensor_helper,
+      "file"_a,
+      "d"_a,
+      py::pos_only(),
+      "retain_graph"_a = true,
```

Comment:
Can you follow the same default as `save` for this? We should in general not retain the graph unless it's a save in a function transform.

See e.g. https://github.com/ml-explore/mlx/pull/215/files#diff-8a607e80673e9f8e9c1e73a5aba9d6f89ac56ad34765e4022d5130f890776cacR2870-R2873